lib/ui/window/platform_configuration_unittests.cc

```diff
@@ -0,0 +1,138 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define FML_USED_ON_EMBEDDER
+
+#include <memory>
+
+#include "flutter/lib/ui/window/platform_configuration.h"
+
+#include "flutter/common/task_runners.h"
+#include "flutter/fml/synchronization/waitable_event.h"
+#include "flutter/lib/ui/painting/vertices.h"
+#include "flutter/runtime/dart_vm.h"
+#include "flutter/shell/common/shell_test.h"
+#include "flutter/shell/common/thread_host.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+class DummyPlatformConfigurationClient : public PlatformConfigurationClient {
+ public:
+  DummyPlatformConfigurationClient() {
+    std::vector<uint8_t> data;
+    isolate_data_.reset(new ::fml::DataMapping(data));
+  }
+  virtual std::string DefaultRouteName() { return "TestRoute"; }
```

Comment:
These should all be marked `override` rather than `virtual`.

Reply:
Good point.  Fixed.